### ISSUE TITLE
Contributions to "New to RSE"

### ISSUE THREAD

**Issue by vsoch** [RSEToolkit/RSEToolkit.github.io#26]
**New content**
Are you requesting or providing new content? _(replace [ ] with [x] to check the relevant box in the list below)_

- [ ] **Content request:** You would like to see new content covering a specific topic added to the resource
- [x] **Providing content:** You are providing some new content for the RSEToolkit (conent to be provided via a pull request)
- [ ] **Other new content-related query:** You have some other query related to content not currently present in the resource (e.g. a question about whether some specific material might be useful/relevant)

**New content information**

Hi! I'm interested in adding the following new/updated content:

- I'm wondering where open source can fit into these docs? E.g., reading about the different [flavors of RSEs](https://rsetoolkit.github.io/roles/new-rse/#what-is-an-rse) the scope is limited to basically provided as a service by a university group vs. embedded in a lab. What about representation for people that provide services via company (e.g., Million Concepts by Chase Million or Dreaming Spires) or RSEng that aren't in a lab or university group but heavily involved in open source development? I'd also want to add to this page something about how students or more traditional roles can be acting as RSEng, and that a graduate degree is common but not required.

I can start here and extend! I think it would be cool if each major org had a link to their "story" - e.g., the UK group already has their link, but US-RSE now has one too :) https://us-rse.org/2022-02-06-a-brief-history-of-usrse/

**Comment by vsoch** [comment]
Also just occurred to me - there might be some overlap between here and https://cosden.github.io/intersect-web/? How could we work together on that? ping @cosden

**Comment by cosden** [comment]
> Also just occurred to me - there might be some overlap between here and https://cosden.github.io/intersect-web/? How could we work together on that? ping @cosden

Thanks @vsoch - I'd be curious how we can collaborate as well. The current project site for INTERSECT (RSE training development and delivery) is here https://intersect-training.github.io/.

**Comment by vsoch** [comment]
Awesome! Please use me as a worker bee if needed, I'm here to help.

**Comment by jcohen02** [comment]
Thanks for these suggestions @vsoch. Definitely happy to include material that recognises other RSE routes/roles/activities. The current "[What is an RSE](https://rsetoolkit.github.io/roles/new-rse/#what-is-an-rse)" material that you linked to feels like it could perhaps be made a bit clearer and more structured. 

While I suppose there are many differently structured RSE roles in place and it's difficult to cover everything, maybe we can have an "Types of RSE role" section or something along these lines that lists various different models including those that you highlight? Perhaps including:

 - University-based - central team
 - University-based - embedded within research group/lab
 - National lab (non-university)
 - Commercial research / R&D-based RSE team
 - Open source research software developers in industry
 - ...?

Not sure how well this fits in with what you had in mind - just a few thoughts but if you're happy to add something along the lines of what you were thinking, that would be fantastic - I'm happy to contribute too.

Linked to this, I'd like to add something along the lines of "_The RSE Continuum_" (tagging @SimonHettrick since we're currently thinking about this) which presents the scale of RSE roles ranging from those at one end that look very similar to industry developers to those at the other end that look very similar to traditional university research roles.

Re @cosden's follow-up, definitely interested to look at opportunities to collaborate - thanks for the link to INTERSECT.

**Comment by vsoch** [comment]
Yes I like that! In my talk I gave recently I talked about "flavors" and they were more akin to the list you provided. Once we hear from @SimonHettrick I can take a shot at updating that page with a listing or continuum of what we decide. Perhaps within each of those groups there is a continuum, or more generally the continuum can be applied to any kind of RSE flavor?

**Comment by jcohen02** [comment]
I think we decided when thinking about this that there are indeed many "continua" in RSE although in a slightly different context to the level of the different RSE flavours....this could end up getting somewhat complex! 🙂
